src/sync/configure.rs

```diff
@@ -32,6 +32,11 @@ fn full_configure(config_asset: &ConfigAsset) -> Option<ToolInfo> {
     let owner = config_asset.owner.clone()?;
     let repo = config_asset.repo.clone()?;
     let exe_name = config_asset.exe_name.clone()?;
+    let tag = config_asset
+        .tag
+        .clone()
+        .map(|version| ToolInfoTag::Specific(version))
```

Comment:
I'm not sure but I believe this simplification should compile 🤔 
```suggestion
        .map(ToolInfoTag::Specific)
```